number/less than tester/f65af71c-28e0-48e2-ad2c-9c8614672002.json

```diff
@@ -1,20 +1,23 @@
 {
-  "description": "Checks if 'a' is less than 'b'.",
+  "description": "Checks if the value received via `a` is less than the value received via `b`. If the values are equal, it outputs false.\n\nExample:\n\n1. 19@0 is received via `a`\n2. 23@0 is received via `b`\n3. true@0 is sent via `less`",
```

Comment:
Maybe rename the ports to value and reference to be more self describing and to be consistent with number/lesser than or equal tester